src/integrationTest/java/com/mongodb/kafka/connect/MongoSourceConnectorTest.java

```diff
@@ -226,24 +236,49 @@ void testSourceCanHandleNonExistentDatabaseAndSurviveDropping() throws Interrupt
             consumer.subscribe(pattern);
 
             MongoDatabase db = getDatabaseWithPostfix();
-            MongoCollection<Document> coll = db.getCollection("coll");
+            MongoCollection<Document> coll1 = db.getCollection("coll1");
```

Comment:
This test case got fleshed out to match the reported one in the ticket (so now uses 3 collections and tests each one).